.github/workflows/release-on-milestone-closed.yml

```diff
@@ -5,8 +5,13 @@ on:
     types:
       - "closed"
 
+permissions:
+  contents: read
+
 jobs:
   release:
+    permissions:
+      contents: none
```

Comment:
Are we really going to be able to create a merge up PR with this permission? And why set a permission globally, only to override it in the one and only job of this file.